.gitignore

```diff
@@ -1,5 +1,7 @@
 .DS_Store
-.pyc
+*~
+*.svg
+*.pyc
```

Comment:
Gotcha, thanks. If it's something that helps vim users, then I see no harm in keeping it.